gymnasium/envs/toy_text/cliffwalking.py

```diff
@@ -17,50 +17,50 @@
 
 class CliffWalkingEnv(Env):
     """
-    This is a simple implementation of the Gridworld Cliff
```

Comment:
Is this an accident?

Reply:
yes, black is splitting the link at line 27 and I'm trying to make flake and black happy without breaking the link